R/build_url.r

```diff
@@ -27,3 +27,4 @@ build_url <- function(company_domain = .get_company_name(),
   url <- glue::glue("{base_url}/{company_domain}/{api_version}")
   return(url)
 }
+

```

Comment:
think this extra line may be spurious, without it, file should match that on develop branch